specification/synapse/resource-manager/Microsoft.Synapse/preview/2019-06-01-preview/privatelinkhub.json

```diff
@@ -21,7 +21,7 @@
     }
   },
   "paths": {
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Synapse/privateLinkHub": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Synapse/privateLinkHubs": {
```

Comment:
This is a breaking change as existing API usage will be broken, please assess the impact.

Reply:
Its a typo fix as the previous API is not existing and wouldn't work

Reply:
@chiragg4u any updates on this?